Source/WebKit/Shared/RemoteLayerTree/CGDisplayList.h

```diff
@@ -35,21 +35,24 @@ class CGDisplayList {
     WTF_MAKE_NONCOPYABLE(CGDisplayList);
 public:
     CGDisplayList() = default;
-    explicit CGDisplayList(WebCore::SharedBuffer& displayList)
+    CGDisplayList(WebCore::SharedBuffer& displayList, Vector<MachSendRight>&& surfaces)
```

Comment:
Maybe we should make a type for a MachSendRight that represents a surface.

Reply:
You mean just a using/typedef?